cfpb/sbl-frontend
# task: Update language for acquiring an LEI

Close #601 

## Changes

- [Filing home] update language
- [Complete user profile] update language

## How to test this PR

1. yarn run dev
2. Visit http://localhost:8899/
3. Confirm language for step "confirm that your financial institution has an LEI" looks good

## Screenshots
|Page|Before|After|
|---|---|---|
|Filing home|![Screenshot 2024-05-29 at 11 54 03 AM](https://github.com/cfpb/sbl-frontend/assets/2592907/12514327-1842-4405-ae94-4cf1756ea6f8)|![Screenshot 2024-05-29 at 11 58 51 AM](https://github.com/cfpb/sbl-frontend/assets/2592907/8892cff7-117f-4670-93e2-d6d8dcf1f1a4)|
